### PR TITLE
Fix code bug

### PR DESCRIPTION
**Pull request by czkqh**
There is an error in the processing of dimensions in some of the forword code:
    cycle_memory = memory[cycle_index, :, :, :]
    cross_cycle_memory = memory[cross_cycle_index, :, :, :]
       ...........
    memory = torch.cat([cycle_memory, cross_cycle_memory], dim=0)

### CODE REVIEW

**Review by Pandalin98 — APPROVED** [review]
fix bug in TT_Fomer.py forward process
